modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/ProposalHashSelect.scala

```diff
@@ -0,0 +1,43 @@
+package org.tessellation.sdk.infrastructure.snapshot
+
+import cats.Monad
+import cats.syntax.functor._
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.{TrustScores, defaultPeerTrustScore}
+import org.tessellation.sdk.domain.snapshot.ProposalSelect
+import org.tessellation.sdk.domain.snapshot.ProposalSelect.occuranceTrustScoreOrdering
+import org.tessellation.sdk.infrastructure.consensus.PeerDeclarations
+import org.tessellation.sdk.infrastructure.snapshot.WeightedSampling.WeightRefinement
+
+import eu.timepit.refined.numeric.Positive
+import eu.timepit.refined.refineV
+
+object ProposalHashSelect {
```

Comment:
shouldn't it be next to the `ProposalSelect`? Will be there any other kind of `ProposalSelect` so you would use the strategy pattern here (unlike the PeerSelect)?

Reply:
Renamed to `ProposalSelect`.